sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuExpandExec.scala

```diff
@@ -170,6 +170,10 @@ class GpuExpandIterator(
             }
             (cv, false)
           case cv: GpuColumnVector => (cv, false)
+          case scalar: Scalar =>
+            withResource(scalar) { s =>
```

Comment:
If the scalar is null, do we want to reuse the null column as if it were the `null` case above?

Reply:
Updated, if the scalar is null, it will go into the last case to create a `ColumnVector ` with a null scalar, the same as the before.

Reply:
The case I met is at line 90 in file "nullExpressions.scala ", where it is trying to extract the value from a cudf Scalar and return. But pulling out list data is difficult, so I change to return a cudf Scalar directly as below.
```diff   
-       GpuScalar.extract(runningScalar)
+      // Returns the Scalar directly instead of pulling data out of GPU.
+      runningScalar.incRefCount()
```